locale/ar_IQ/currencies.xml

```diff
@@ -0,0 +1,176 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE currencies SYSTEM "../../dtd/currencies.dtd">
+
+<!--
+  * locale/ar_IQ/currencies.xml
+  *
+  * Copyright (c) 2013-2016 Simon Fraser University Library
+  * Copyright (c) 2000-2016 John Willinsky
+  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+  *
+  * Currencies data XML file. Based on ISO 4217.
+  -->
+
+<currencies locale="ar_IQ">
```

Comment:
Yes, other currencies are barely known to the average Arabic reader, so I left them untranslated.